README.md

```diff
@@ -124,6 +124,29 @@ if ((brushedTeeth || tookMints) && didHomework && preparedForRobotics) {
  - What happens?
  - Ok, so working -1 hours and then stopping is not realistic, but this code does show as a check to make sure that something unrealistic doesn't happen, and prevents it becoming a ***major*** problem.
 
+### For Loops
+ 1. Open For.java and read through the code.
+ - The code for `for` loops looks a little confusing, but it makes sense when you break it down.
+     - `for (int i = 0; i < 20; i++) { }` breaks down into:
+ ```java
+ int i = 0;
+ while(i < 20){
+    // Code inside for loop here!
+    i++;
+ }
+ ```
+ - Yep, a for loop is only a disguised while loop!
```

Comment:
This isn't completely true. Both are loops but one iterates the iterator each loop (for loop), while another just loops until a condition is met. (while loop) Now you can setup a while loop to iterate on an iterator like a for loop but its not required unlike in the for loop.

I would re-word this real quick